TestHarness/Cucumber/features/ScenarioOutlines.feature

```diff
@@ -67,3 +67,13 @@ Scenario Outline: Deal correctly with backslashes in the examples
   Examples:
     | file path |
     | c:\Temp\  |
+
+
+Scenario Outline: Deal correctly with parenthesis in the examples
+
+  When I have parenthesis in the value, for example a '<overly descriptive field>'
+
+  Examples:
+    | overly descriptive field		   |
```

Comment:
In order to increase legibility, please format the table so that the borders of the table line up.